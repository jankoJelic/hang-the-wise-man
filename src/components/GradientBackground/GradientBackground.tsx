import useDimensions from "hooks/useDimensions";

type Props = {
  children: React.ReactNode;
};
const GradientBackground: React.FC<Props> = ({ children }) => {
  const { width, height } = useDimensions();

  return (
    <main
      style={{ minWidth: width, minHeight: height }}
      className="flex flex-col items-center pt-10 bg-gradient-to-br from-white to-mainExtraLight"
    >
      {children}
    </main>
  );
};

export default GradientBackground;
