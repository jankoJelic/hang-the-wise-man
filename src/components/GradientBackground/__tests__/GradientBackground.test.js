import renderer from "react-test-renderer";
import GradientBackground from "../GradientBackground";

it("renders correctly", () => {
  const tree = renderer.create(<GradientBackground />).toJSON();
  expect(tree).toMatchSnapshot();
});
