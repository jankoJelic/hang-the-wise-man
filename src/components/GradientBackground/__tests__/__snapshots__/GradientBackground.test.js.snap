// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  className="flex flex-col items-center pt-10 bg-gradient-to-br from-white to-mainExtraLight"
  style={
    Object {
      "minHeight": 0,
      "minWidth": 0,
    }
  }
/>
`;
